best_practices.md

```diff
@@ -0,0 +1,195 @@
+# Best practices for programming diagnostics
+
+To facilitate code integration, code sharing between developers, reusability, easy
+maintenance, and readability, it is recommended that these recommendations
+are followed when developing new diagnostic scripts.
+
+## Make your code open-source
+
+It is recommended that you make your code
+[open-source](https://the-turing-way.netlify.app/reproducible-research/open/open-source.html)
+from the start, to facilitate reproducible science and encourage potential
+[collaboration](https://the-turing-way.netlify.app/collaboration/collaboration.html).
+If you have reservations about making your code open source before publication,
+consider developing it in a private [git repository](#use-git-for-version-control) and making it
+public once you have submitted your paper.
+
+## Add a license
```

Comment:
Is there a default license or recommendation? Like Apache 2.0 ...

Reply:
The recommendation is to use whatever is the most common choice in the community you're working in. I have update the text accordingly